nucypher/blockchain/eth/sol/source/contracts/ChallengeOverseer.sol

```diff
@@ -27,7 +26,9 @@ contract ChallengeOverseer {
     constructor(
         MinersEscrow _escrow,
         SignatureVerifier.HashAlgorithm _hashAlgorithm
-    ) public {
+    )
+        public
+    {
```

Comment:
Is this really more readable? haha

Reply:
To have a single style, in other places this looks much better